### PR TITLE
Added info about related to the silent release of NIM

### PR DESCRIPTION
**Pull request by Reinmar**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Added info about related to the silent release of NIM.

---

### Additional information

See: https://github.com/ckeditor/ckeditor5/issues/16360#issuecomment-2112148609

### CODE REVIEW

**Review by Witoso — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ We have also added [menu bar](https://ckeditor.com/docs/ckeditor5/latest/feature
 ### MINOR BREAKING CHANGES [ℹ️](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/support/versioning-policy.html#major-and-minor-breaking-changes)
 
 * **[ui](https://www.npmjs.com/package/@ckeditor/ckeditor5-ui)**: The region name argument of the `AriaLiveAnnouncer#announce()`  method has been dropped. Please check out the latest API documentation for more information.
+* The `ckeditor5` package now lists all other official `@ckeditor/ckeditor5-*` packages as dependencies. This is a preparatory step for the upcoming [new installation methods](https://github.com/ckeditor/ckeditor5/issues/15502). These changes will transform the `ckeditor5` package into an aggregate for all official packages, simplifying module imports.
```
Comment (pomek) [inline CHANGELOG.md:15]:
```suggestion
* The `ckeditor5` package now lists all other official open-source `@ckeditor/ckeditor5-*` packages as dependencies. This is a preparatory step for the upcoming [new installation methods](https://github.com/ckeditor/ckeditor5/issues/15502). These changes will transform the `ckeditor5` package into an aggregate for all official packages, simplifying module imports.
```

**Comment by bnf** [comment]
As asked in https://github.com/ckeditor/ckeditor5/issues/16360 – will your refactor the codebase to make the `ckeditor5` package optional, importing from `@ckeditor/ckeditor5-*` directly instead of `ckeditor5/src/`?

With this change `ckeditor5` package got two roles:

 * central package for all packages for engine/library methods
 * bundle for all opens-ource packages

That means every `@ckeditor/ckeditor5-*` that depends on  `ckeditor5` will pull in every other open-source `@ckeditor/ckeditor5-*` package via the transitive dependency through `ckeditor5`.

**Comment by Witoso** [comment]
As far as I know, this cannot happen yet, as we will continue to support old installation methods for a while. This refactor will occur once the deprecation timeline for old methods is complete.

**Pull request merged**